### ISSUE TITLE
Can't Install Fresh Clone

### ISSUE THREAD

**Issue by LoganGrier** [aantron/bisect_ppx#403]
## Command
```
bash src/npm-install.sh
```
## Console Output
```
bin/linux/ppx not found; falling back to a source build.
+ esy install -P binaries.esy.json
info install 0.6.12-dev (using binaries.esy.json)
info checking https://github.com/ocaml/opam-repository for updates...
info checking https://github.com/esy-ocaml/esy-opam-override for updates...
info resolving esy packages: done                           
info solving esy constraints: done         
info resolving npm packages: done
info fetching: done                                 
info installing: done                                          
+ esy -P binaries.esy.json dune build -p bisect_ppx src/ppx/js/ppx.exe
(cd _build/default && /home/lgrier/.esy/3__________________________________________________________________/i/ocaml-4.14.0-c4de7b44/bin/ocamlopt.opt -w -40 -open Ocaml_shadow -g -I src/ppx/.bisect_ppx.objs/byte -I src/ppx/.bisect_ppx.objs/native -I /home/lgrier/.esy/3__________________________________________________________________/i/ocaml-4.14.0-c4de7b44/lib/ocaml/compiler-libs -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__ocaml_compiler_libs-opam__c__v0.12.4-2dd80bec/lib/ocaml-compiler-libs/common -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__ocaml_compiler_libs-opam__c__v0.12.4-2dd80bec/lib/ocaml-compiler-libs/shadow -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__ppx__derivers-opam__c__1.2.1-cde3bc8f/lib/ppx_derivers -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__ppxlib-opam__c__0.27.0-ba139a4b/lib/ppxlib -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__ppxlib-opam__c__0.27.0-ba139a4b/lib/ppxlib/ast -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__ppxlib-opam__c__0.27.0-ba139a4b/lib/ppxlib/astlib -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__ppxlib-opam__c__0.27.0-ba139a4b/lib/ppxlib/print_diff -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__ppxlib-opam__c__0.27.0-ba139a4b/lib/ppxlib/stdppx -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__ppxlib-opam__c__0.27.0-ba139a4b/lib/ppxlib/traverse_builtins -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__sexplib0-opam__c__v0.15.1-15a548e8/lib/sexplib0 -I /home/lgrier/.esy/3__________________________________________________________________/i/opam__s__stdlib_shims-opam__c__0.3.0-1152838f/lib/stdlib-shims -I src/common/.bisect_common.objs/byte -I src/common/.bisect_common.objs/native -intf-suffix .ml -no-alias-deps -open Bisect_ppx -o src/ppx/.bisect_ppx.objs/native/bisect_ppx__Instrument.cmx -c -impl src/ppx/instrument.pp.ml)
File "src/ppx/instrument.ml", line 494, characters 29-31:
494 |         recur ~enclosing_loc p'
                                   ^^
Error: This expression has type
         string Astlib.Ast_414.Asttypes.loc list * Parsetree/1.pattern
       but an expression was expected of type
         Parsetree/1.pattern = Parsetree/2.pattern
       File "src/ppx/instrument.ml", line 37, characters 0-35:
         Definition of module Parsetree/1
       File "_none_", line 1:
         Definition of module Parsetree/2
```
## Commit Id
```
> git rev-parse HEAD
816def34eacfa0d12a6fd53347f230d9a997a4cc
```
## System Information
```
> ocaml --version
The OCaml toplevel, version 4.14.0
> dune --version
3.3.1
> rescript --version
bash: rescript: command not found
> cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
> uname -r
5.4.72-microsoft-standard-WSL2
```

**Comment by roptat** [comment]
In case this is still helpful, it seems you're using ppxlib 0.27, but bisect_ppx is only compatible with bisect_ppx until 0.25 (included). You'll have to downgrade ppxlib or wait for #400 to be merged.

**Comment by aantron** [comment]
#400 has been merged and will be included in the next release, Bisect_ppx 2.8.2.
